docs/content/overview.md

```diff
@@ -493,6 +494,7 @@ upon backend-specific capabilities.
 | Microsoft OneDrive           | Yes   | Yes  | Yes  | Yes     | Yes     | No    | No           | Yes          | Yes   | Yes      |
 | OpenDrive                    | Yes   | Yes  | Yes  | Yes     | No      | No    | No           | No           | No    | Yes      |
 | OpenStack Swift              | Yes † | Yes  | No   | No      | No      | Yes   | Yes          | No           | Yes   | No       |
+| Oracle Object Storage        | Yes   | Yes  | No   | No      | Yes     | Yes   | No           | No           | No    | No       |
```

Comment:
Are you sure it is correct to say `CleanUp=Yes`, according to https://rclone.org/overview/#cleanup?

Reply:
Also, judging from https://github.com/rclone/rclone/blob/d2fef05fe45074c458ae11d6b3361a2bd26c9c10/backend/oracleobjectstorage/oracleobjectstorage.go#L682-L694 it seems, in addition to `CleanUp=No`, then it should also have `Purge=No` and `StreamUpload=Yes`, unless I'm missing something here?

Reply:
It does have Cleanup support, but the interface test is missing.

It doesn't have Purge support and does have PutStream support.